.github/workflows/deploy_to_aws.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+      - deploy-*
```

Comment:
Was this branch added for testing purposes?

Reply:
Yes, and I have an idea to keep it for further purpose. But if you find any difficulties with it, I remove this from PR.

Reply:
Actually, I'm not entirely certain about the challenges involved. However, to ensure clarity for anyone working on this workflow and to prevent confusion with unrelated branches, I would suggest removing it.

Reply:
If you have any plans for this branch, let's leave it.

Reply:
Testing is exactly what  I wanted to achieve: to have an approach how to test new deployment features

Reply:
@vladsha-dev could you please resolve conversation if you okay. I few see why this line can be dangerous please bring it here

Reply:
I would add a comment stating that this branch is for testing new deployment features.